frame/timestamp/Cargo.toml

```
@@ -44,6 +45,7 @@ std = [
 	"sp-io?/std",
 	"sp-runtime/std",
 	"sp-std/std",
+  "sp-storage/std",
```

```suggestion
	 "sp-storage/std",
```
CI noticed :sunglasses: